ros_buildfarm/templates/snippet/reconfigure_jobs.groovy.em

```diff
@@ -61,7 +61,7 @@ if (views.size() != @(expected_num_views)) {
     throw new AbortException("Wrong number of view configs")
 }
 
-views.each{
+for (it in views) {
```

Comment:
It is really odd that the iterator was not resolving. But at least the for loop works.